.editorconfig

```diff
@@ -0,0 +1,31 @@
+# EditorConfig is awesome: http://EditorConfig.org
+
+# top-most EditorConfig file
+root = true
+
+# Unix-style newlines with a newline ending every file
+[*]
+end_of_line = lf
+insert_final_newline = true
+trim_trailing_whitespace = true
+max_line_length = 120
```

Comment:
`end_of_line = lf` and `max_line_length = 120` are found at [Code Style](https://github.com/spring-projects/spring-framework/wiki/Spring-Framework-Code-Style). But I also find some lines are more than 120 characters. 

The other two are found in codes.

How do I change it as follows:

```
end_of_line = lf
insert_final_newline = true
trim_trailing_whitespace = true
```